Headers/AppKit/NSAppearance.h

```diff
@@ -0,0 +1,83 @@
+/* Definition of class NSAppearance
+   Copyright (C) 2019 Free Software Foundation, Inc.
+   
+   By: Gregory John Casamento
+   Date: Wed Jan 15 07:03:39 EST 2020
+
+   This file is part of the GNUstep Library.
+   
+   This library is free software; you can redistribute it and/or
+   modify it under the terms of the GNU Lesser General Public
+   License as published by the Free Software Foundation; either
+   version 2.1 of the License, or (at your option) any later version.
+   
+   This library is distributed in the hope that it will be useful,
+   but WITHOUT ANY WARRANTY; without even the implied warranty of
+   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+   Library General Public License for more details.
+   
+   You should have received a copy of the GNU Lesser General Public
+   License along with this library; if not, write to the Free
+   Software Foundation, Inc., 51 Franklin Street, Fifth Floor,
+   Boston, MA 02110 USA.
+*/
+
+#ifndef _NSAppearance_h_GNUSTEP_GUI_INCLUDE
+#define _NSAppearance_h_GNUSTEP_GUI_INCLUDE
+
+#import <Foundation/NSObject.h>
+#import <Foundation/NSBundle.h>
+
+#if OS_API_VERSION(MAC_OS_X_VERSION_10_9, GS_API_LATEST)
+
+#if	defined(__cplusplus)
+extern "C" {
+#endif
+
+typedef NSString* NSAppearanceName;
+
+@interface NSAppearance : NSObject <NSCopying, NSCoding>
```

Comment:
You're correct, I will remove this.

> > I'm implementing this mainly for compatibility, but given NSAppearance has some similarity to our theme system I am wondering if there needs to be some bridging done here. Please put your comments here.
> 
> I don't thin[k] this directly maps to our GSTheme class. That would rather correspond to TDThemeSchema. NSAppearance looks like an abstraction on top of that.

I believe you're correct.  It feels like more of a way of SETTING an "appearance" than a representation of an appearance.   For the time being I will leave it as a stub, but it might be worth revisiting at some point in the future.  Do you agree with that opinion?

Reply:
Removed NSCopying implementation.